Assets/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "com.microsoft.mixedrealitytoolkit",
+  "version": "2.0.0",
+  "displayName": "Mixed Reality Toolkit",
+  "description": "MRTK virtal package.",
```

Comment:
```suggestion
  "description": "MRTK virtual package.",
```

Reply:
Thanks for catching this @keveleigh. This looks like a great fix and I'm trying to accept it, but I'm getting this error:

![image](https://user-images.githubusercontent.com/5230670/60603775-ff396300-9d7b-11e9-9b20-7283ab5322c4.png)

I'm wondering if it's because of where I created the branch? When I went to commit this branch to source control I got an error about needing to do a pull request instead, but all of the files were actually pushed up into the branch. I'm wondering if this error about not being able to apply the suggestion is somehow due to a DevOps rule?

Reply:
Confirmed. This is because it's a "protected branch".

![image](https://user-images.githubusercontent.com/5230670/60604439-4e33c800-9d7d-11e9-880f-bb2df47c3dc2.png)

@keveleigh or @davidkline-ms any idea why I was able to create this branch and push to it, but now I can't update it? I see lots of feature branches. Do I just not have the right permissions as a contributor or how should we move forward?